prebid-mobile/prebid-mobile.md

```diff
@@ -11,31 +11,93 @@ nav_section: prebid-mobile
 <div class="bs-docs-section" markdown="1">
 
 # Prebid Mobile Overview
-{:.no_toc}
 
-Prebid Mobile provides libraries for Android and iOS that provide an end-to-end open-source header bidding solution for mobile app publishers.
+* TOC
+{:toc }
 
-Benefits and features include:
+Prebid Mobile helps app publishers by making the advantages of “traditional” web header bidding available to mobile app environments. This is accomplished by providing libraries that enable mobile app publishers to implement end-to-end, open source header bidding solutions.
 
-- Single integration point with Prebid Server with direct access to more mobile buyers
-- Server side configuration management for bidders and other settings, no need to update your app to make changes
-- Minimized latency due to background pre-caching of creatives
-- Increased yield due to increased competition between demand sources
-- A lightweight library that uses simple query strings to send ad requests and to pass results
+## Benefits and Features
 
-### How it Works
+Some of the benefits to using the Prebid Mobile header bidding solution include:
 
-The Prebid Mobile SDK sends requests to Prebid Server to help app developers to access additional demand that is not available from their mobile ad network.
+-   Provides a single integration point with Prebid Server, providing direct access to more mobile buyers.
+-   Server-side configuration management; no need for developers to update the app to make configuration changes.
+-   Prebid Mobile is a transparent, open source solution.
+-   Created for integration with other prebid products and open source servers.
```

Comment:
Took that from these notes: https://corpwiki.appnexus.com/display/~msudberg/Prebid+Mobile+Overview
Maybe I misunderstood what the note was talking about, but I can take this line out.

Reply:
Prebid Mobile is designed to integrate with Prebid Server.  It can integrate with any vendor hosting their own Prebid Server deployment as long as that vendor is registered in Prebid Mobile as a Prebid Server host.